conans/requirements.txt

```diff
@@ -1,4 +1,5 @@
-PyJWT>=1.4.0, <2.0.0
+PyJWT>=1.4.0, <2.0.0; python_version < '3.0'
+PyJWT>=2.0.0, <3.0.0; python_version >= '3.0'
```

Comment:
Maybe better maintain PyJWT>=1.4.0

Reply:
have to check if prefers the first, but yes.